### PR TITLE
Created separate module for rand, updated Raffle contract to use it.

### PR DESCRIPTION
**Pull request by mariopil**
**Referenced issue**

https://github.com/eigerco/nebula/issues/84

### CODE REVIEW

File: sample-contract/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+#![no_std]\nuse soroban_sdk::{contract, contractimpl, symbol_short, vec, Env, Symbol, Vec};\n\n#[contract]\npub struct HelloContract;\n\n#[contractimpl]\nimpl HelloContract {\npub fn hello(env: Env, to: Symbol) -> Vec<Symbol> {\nvec![&env, symbol_short!(\"Hello\"), to]\n}\n}
```
Comment (eloylp) [inline sample-contract/src/lib.rs:1]: What is this ? :)

**Review by eloylp — APPROVED** [review]

**Comment by eloylp** [comment]
Some lint errors were detected : https://github.com/eigerco/nebula/actions/runs/6585950934/job/17893341443?pr=93

File: sample-contract/src/lib.rs (hunk 1 above)
Comment (mariopil) [reply sample-contract/src/lib.rs:1]: I think there's a problem with line ending... Let me see if I can fix this.

Comment (mariopil) [reply sample-contract/src/lib.rs]: Ok, I think I've messed up when something, this shouldn't be there :)

Comment (eloylp) [reply sample-contract/src/lib.rs]: ok ! no prob. marking as resolved.

**Pull request merged**